src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -128,26 +128,34 @@ void EquationInputArea::InputTextBox_Submitted(Object ^ sender, EquationSubmissi
 
```

Comment:
Not sure if related to this PR, but I see a crash in the app on the above line 123 in this scenario:

1. Type a valid equation into the textbox and submit
2. Click the f(1) button to hide the equation
3. Click remove button to remove the hidden equation
4. Crash from out of bounds exception

Reply:
fixed!